VideoLocker/src/main/java/org/edx/mobile/model/course/CourseComponent.java

```diff
@@ -37,6 +38,16 @@ public CourseComponent(){}
      * @param parent  is null if and only if this is the root
      */
     public CourseComponent(BlockModel blockModel, CourseComponent parent){
+        this(blockModel, parent, false);
```

Comment:
Also, I'm not really sure it makes sense to have this passed as an argument. Shouldn't we be able to infer it from the block model?